WordPress/Classes/ViewRelated/Cells/MediaSizeSliderCell.swift

```diff
@@ -0,0 +1,143 @@
+import Foundation
+import UIKit
+
+class MediaSizeSliderCell: UITableViewCell {
+    // MARK: - Default values
+    struct Default {
```

Comment:
Not sure I follow. Do you mean having `MediaSizeSliderCell.Model.default` return a model with default values?
That could work, although I kind of like having the default values all grouped at the top of the file.